src/lib/components/CourseLinks/index.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 Google LLC
+ * Copyright 2023 Google LLC
```

Comment:
From what I am used to I wouldn't update the year in the license header, as I _think_ it reads like 'from that point on'. But might well be I am missing out on some kind of policy or guideline.

Reply:
Looks like you're right! Reverted it.